STATUS: INLINE FAIL
MODIFIED: pitt.search.semanticvectors.vectors.ComplexVectorTest.testComplexOrthogonalize
MODIFIED FILE: /home/lichengpeng/test/automated_script/Lucky-Dhakad/semanticvectors/src/test/java/pitt/search/semanticvectors/vectors/ComplexVectorTest.java
CLEANER: pitt.search.semanticvectors.integrationtests.RegressionTests.testBuildAndSearchBasicComplexIndex
CLEANER FILE: /home/lichengpeng/test/automated_script/Lucky-Dhakad/semanticvectors/src/test/java/pitt/search/semanticvectors/integrationtests/RegressionTests.java
POLLUTER: pitt.search.semanticvectors.CompareTermsTest.testCompareTermsComplexFlatOrthographic
POLLUTER FILE: /home/lichengpeng/test/automated_script/Lucky-Dhakad/semanticvectors/src/test/java/pitt/search/semanticvectors/CompareTermsTest.java
ORIGINAL CLEANER SIZE: 5
NEW CLEANER SIZE: 1
ELAPSED TIME: 20.187000036239624
==========================
@@ -224,0 +224,3 @@
+ {
+     assertEquals(2, buildSearchGetRank("-dimension 200 -vectortype complex -luceneindexpath positional_index", "peter", "simon"));
+ }
